dev-client/package.json

```diff
@@ -29,7 +29,7 @@
     "react-native-app-auth": "^6.4.3",
     "react-native-autocomplete-input": "^5.3.2",
     "react-native-config": "^1.5.1",
-    "react-native-gesture-handler": "^2.12.1",
+    "react-native-gesture-handler": "software-mansion/react-native-gesture-handler#main",
```

Comment:
suggestion: we should pin the current commit from main so we don't accidentally pull in broken changes